### PR TITLE
README: update, emphasize community importance

### PR DESCRIPTION
**Pull request by Andersson007**
README: update, emphasize community importance

### CODE REVIEW

**Comment by Andersson007** [comment]
thanks @leogallego for spotting things that can be improved!

**Comment by Andersson007** [comment]
ready for review

File: README.md [hunk 1]
```diff
@@ -2,18 +2,13 @@
 
 [![Discuss on Matrix at #community:ansible.com](https://img.shields.io/matrix/community:ansible.com.svg?server_fqdn=ansible-accounts.ems.host&label=Discuss%20on%20Matrix%20at%20%23community:ansible.com&logo=matrix)](https://matrix.to/#/#community:ansible.com)
 
-This repository contains topics that will be discussed and voted by the Ansible Community and the [Ansible Community Steering Committee](https://docs.ansible.com/ansible/devel/community/steering/community_steering_committee.html) asynchronously.
+This repository contains topics that will be discussed and voted by the Ansible Community (all interested people) and the [Ansible Community Steering Committee](https://docs.ansible.com/ansible/devel/community/steering/community_steering_committee.html) asynchronously.
 
-If you want to discuss an idea, improvements or submit new Policy/Proposals & New Collection Inclusion Requests then create a new [issue](https://github.com/ansible-community/community-topics/issues) in this repo as a topic.
+The **scope** of the topics is changes impacting the Ansible community, contributor experience, Ansible Community package and community collections.
```
Comment (felixfontein) [inline README.md:7]:
```suggestion
The **scope** of the topics is changes impacting the Ansible community, contributor experience, Ansible Community package, and community collections.
```

File: README.md [hunk 2]
```diff
@@ -2,18 +2,13 @@
...
 
-## Process for handling meeting topics
+If you want to discuss something that satisfies the scope then create a new [topic](https://github.com/ansible-community/community-topics/issues) or comment on [existing](https://github.com/ansible-community/community-topics/issues) one.
```
Comment (felixfontein) [inline README.md:9]:
```suggestion
If you want to discuss something that satisfies the scope then [create a new topic](https://github.com/ansible-community/community-topics/issues) or [comment on an existing one](https://github.com/ansible-community/community-topics/issues).
```

**Comment by Andersson007** [comment]
@felixfontein , incorporated, thanks, PTAL

**Comment by felixfontein** [comment]
@leogallego @Andersson007 thanks for improving the README!

**Pull request merged**